src/components/Charts.tsx

```diff
@@ -178,10 +177,10 @@ export default function ChartViewer({
   );
 
   // Calculate dataset totals
-  for (const transaction of cleanTransactions) {
-    chartLinesDataset[transaction.monthIndex]["dataset"][
-      transaction.groupName
-    ] += transaction.amount;
+  for (const [, { month, group, amount }] of Array.from(
+    allTransactions.transactions,
+  )) {
+    chartLinesDataset[month]["dataset"][group] += amount;
```

Comment:
Je ferais plutôt une view dans le MST qui calcule ça plutôt que de le lier à l'UI

Reply:
Moved to MST in PR #10